Dfe.PrepareConversions/Dfe.PrepareConversions.Data/Features/ApiClient.cs

```diff
@@ -14,56 +11,39 @@ public class ApiClient : IApiClient
 {
    private readonly IDfeHttpClientFactory _httpClientFactory;
    private readonly PathFor _pathFor;
-   private readonly bool _useAcademisation;
    private readonly bool _useAcademisationApplication;
 
    public ApiClient(IDfeHttpClientFactory httpClientFactory, IFeatureManager features, PathFor pathFor)
    {
       _pathFor = pathFor;
       _useAcademisationApplication = features.IsEnabledAsync(FeatureFlags.UseAcademisationApplication).Result;
       _httpClientFactory = httpClientFactory;
-      _useAcademisation = features.IsEnabledAsync(FeatureFlags.UseAcademisation).Result;
    }
 
    private HttpClient TramsClient => _httpClientFactory.CreateTramsClient();
    private HttpClient AcademisationClient => _httpClientFactory.CreateAcademisationClient();
-   private HttpClient ActiveClient => _useAcademisation ? AcademisationClient : TramsClient;
    private HttpClient ActiveApplicationClient => _useAcademisationApplication ? AcademisationClient : TramsClient;
 
    public async Task<HttpResponseMessage> GetAllProjectsAsync(AcademyConversionSearchModel searchModel)
    {
-      return await ActiveClient.PostAsync(_pathFor.GetAllProjects, JsonContent.Create(searchModel));
+      return await AcademisationClient.PostAsync(PathFor.GetAllProjects, JsonContent.Create(searchModel));
    }
 
 
    public async Task<HttpResponseMessage> GetProjectByIdAsync(int id)
    {
-      Task<HttpResponseMessage> getProjectResponse = ActiveClient.GetAsync(string.Format(_pathFor.GetProjectById, id));
-      Task<HttpResponseMessage> getProjectNotesResponse =
-         _useAcademisation ? Task.FromResult(new HttpResponseMessage()) : TramsClient.GetAsync(string.Format(PathFor.GetProjectNotes, id));
-
-      await Task.WhenAll(getProjectResponse, getProjectNotesResponse);
-
-      if (_useAcademisation) return getProjectResponse.Result;
-      if (getProjectResponse.Result.IsSuccessStatusCode is false) return getProjectResponse.Result;
-
-      AcademyConversionProject project = await getProjectResponse.Result.Content.ReadFromJsonAsync<AcademyConversionProject>();
-      if (project is null) return new HttpResponseMessage(HttpStatusCode.NotFound);
-
-      if (getProjectNotesResponse.Result.IsSuccessStatusCode)
```

Comment:
We're no longer getting the ProjectNotes, is that intended?

Reply:
Whilst the old code might look like it was getting notes with a separate Api call it actually wasn't if the flag was set.
The old code would exit on line 46, but only set the project.Notes property on line 53

![image](https://github.com/DFE-Digital/prepare-academy-conversions/assets/109349584/f388cfcf-1745-486a-830c-4481a1a79e33)

Reply:
That is very strange, I wonder if that's a hangover for when we were porting the endpoints across as notes were last to come over, I think they probably should've been returned here but since they weren't anyway I'll resolve this and maybe we make a note of this to check/test notes are still able to be got etc

Reply:
Notes still come through, they must be part of the main entity being returned
![image](https://github.com/DFE-Digital/prepare-academy-conversions/assets/109349584/964abdeb-3001-44b5-bb2a-daf05b868894)